vscode-lean4/src/utils/localStorage.ts

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+// Copied from this awesome blog:
+// https://www.chrishasz.com/blog/2020/07/28/vscode-how-to-use-local-storage-api/
+
+import { Memento } from 'vscode';
+
+export class LocalStorageService {
```

Comment:
I suppose we could - I copied this pattern from the blog I read, but you're right the class doesn't really add anything, perhaps it could be a handy place to intercept all local storage for debugging, or for also adding default values in the future...